### PR TITLE
Created ontology for customer

### PR DESCRIPTION
**Pull request by janbj**

### CODE REVIEW

File: src/customer-ontology.ttl [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+  rdfs:comment "Customer institution in NVA" .
+
+:VocabularySetting a rdf:Class ;
+  rdfs:comment "Setting for a controlled vocabulary for a customer institution" .
+
+:VocabularyStatus a rdf:Class ;
+  rdfs:comment "Status for a controlled vocabulary for a customer institution" .
+
+:name a rdf:Property ;
+  rdfs:comment "The name of the customer institution" ;
```
Comment (axthosarouris) [inline src/customer-ontology.ttl:20]: The descriptions of the properties are quite short and they basically rephrase the the name of the property. In the cases where the explanation cannot be more detailed, can we add an example value?

File: src/customer-ontology.ttl [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+
+# Customer-related concepts
+
+:Customer a rdf:Class ;
+  rdfs:comment "Customer institution in NVA" .
+
+:VocabularySetting a rdf:Class ;
+  rdfs:comment "Setting for a controlled vocabulary for a customer institution" .
+
+:VocabularyStatus a rdf:Class ;
```
Comment (axthosarouris) [inline src/customer-ontology.ttl:16]: Is VocuabularyStatus an enumeration?  If yes, the values of the enumeration could be subclasses of the enumeration class.

**Review by axthosarouris — CHANGES_REQUESTED** [review]

File: src/customer-ontology.ttl (hunk 1 above)
Comment (brinxmat) [reply src/customer-ontology.ttl:20]: It's typically the case that a vocabulary includes an rdfs:label, which repeats the token (so here, rdf:label "name"). For the comment, it's hard to find an alternative for name, but I'd suggest, e.g. "A string representing what the institution is called"

File: src/customer-ontology.ttl [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+  rdfs:comment "Status for a controlled vocabulary for a customer institution" .
+
+:name a rdf:Property ;
+  rdfs:comment "The name of the customer institution" ;
+  rdfs:domain :Customer ;
```
Comment (brinxmat) [inline src/customer-ontology.ttl:21]: rdfs:isSubPropertyOf rdfs:label would help using this file in a semantic sense.

File: src/customer-ontology.ttl [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+@prefix owl: <http://www.w3.org/2002/07/owl#> .
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix schema: <https://schema.org/> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+
+# Customer-related concepts
+
+:Customer a rdf:Class ;
+  rdfs:comment "Customer institution in NVA" .
```
Comment (brinxmat) [inline src/customer-ontology.ttl:11]: Technically, every property and class should have rdfs:isDefinedBy customer:, if that makes sense.

File: src/customer-ontology.ttl [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+:VocabularySetting a rdf:Class ;
+  rdfs:comment "Setting for a controlled vocabulary for a customer institution" .
+
+:VocabularyStatus a rdf:Class ;
+  rdfs:comment "Status for a controlled vocabulary for a customer institution" .
+
+:name a rdf:Property ;
+  rdfs:comment "The name of the customer institution" ;
+  rdfs:domain :Customer ;
+  rdfs:range xsd:string .
```
Comment (brinxmat) [inline src/customer-ontology.ttl:22]: Check if the ranges are xsd:string or rdf:langString for all string props

File: src/customer-ontology.ttl (hunk 3 above)
Comment (brinxmat) [reply src/customer-ontology.ttl:21]: This is probably the case for every label-like property

**Review by brinxmat — CHANGES_REQUESTED** [review]

File: src/customer-ontology.ttl (hunk 5 above)
Comment (janbj) [reply src/customer-ontology.ttl:22]: They are all xsd:string right now, but maybe some really should be rdf:langString?

File: src/customer-ontology.ttl [hunk 6]
```diff
@@ -0,0 +1,122 @@
...
+
+customer:created a rdf:Property ;
+  rdfs:comment "A point in time when a resource was first created" ;
+  rdfs:isDefinedBy customer: ;
+  rdfs:domain rdfs:Class ;
+  rdfs:range xsd:dateTime ;
+  rdfs:subPropertyOf dcterms:created .
+
+customer:modified a rdf:Property ;
+  rdfs:comment "A point in time when a resource was last changed" ;
```
Comment (Eirik-Eide-Saur) [inline src/customer-ontology.ttl:118]: Should the format of the dateTime be specified?

Comment (janbj) [reply src/customer-ontology.ttl:118]: rdfs:range xsd:dateTime ;

Comment (Eirik-Eide-Saur) [reply src/customer-ontology.ttl:118]: Is that allways yyyy-mm-dd?

Comment (brinxmat) [reply src/customer-ontology.ttl:118]: http://books.xmlschemata.org/relaxng/ch19-77049.html

Comment (brinxmat) [reply src/customer-ontology.ttl:118]: Note that there may be an issue with time zones, but…I have to admit I am not sure

File: src/customer-ontology.ttl [hunk 7]
```diff
@@ -0,0 +1,122 @@
...
+customer:archiveName a rdf:Property ;
+  rdf:label "archiveName" ;
+  rdfs:comment "A name used when describing the set of data owned by an institution" ;
+  rdfs:isDefinedBy customer: ;
+  rdfs:isSubPropertyOf rdfs:label ;
+  rdfs:domain customer:Customer ;
+  rdfs:range xsd:string .
+
+customer:cname a rdf:Property ;
+  rdfs:comment "A Canonical Name (DNS) that should resolve the set of data owned by an institution" ;
```
Comment (brinxmat) [inline src/customer-ontology.ttl:67]: Is there a reason why no labels on the following props?

**Review by brinxmat — CHANGES_REQUESTED** [review]

**Review by axthosarouris — APPROVED** [review]

**Review by Eirik-Eide-Saur — APPROVED** [review]

**Review by brinxmat — APPROVED** [review]

**Pull request merged**